.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: erlang
+dist: trusty
```

Comment:
part of the patch have already been merged via another one but i'm curious about that change. Why changing the distribution?

Reply:
?

Reply:
Looking the [Travis's document](https://docs.travis-ci.com/user/languages/erlang/), Travis doesn't support OTP 20.3 on Xenial or later. So to run tests on erlang-20.3, we should use trusty.

Now, hackney is not tested on erlang-20.3. It's meaningless to change the distribution.